godotproject/rat_spritesheet.png.import

```diff
@@ -2,15 +2,15 @@
 
 importer="texture"
 type="StreamTexture"
-path="res://.import/birds.png-e63708046b8c22ce4989dac9da8485df.stex"
+path="res://.import/rat_spritesheet.png-0ef4b8f8dd66edc1ec1cb2e54751a5e7.stex"
```

Comment:
Git seems to have marked this as replacing the `birds.png.import` file, which I don't think is correct and will probably break the bird. Pls give it another check as to whether this is intended and whether everything still works